raglweb/__init__.py

```diff
@@ -45,6 +45,12 @@ def _db_close(e=None):
         db.close()
 
 
+def _ordered_alphanumerically(x: str, y: str) -> bool:
+    # returns the alphanumerically higher-ranked string, i.e. "abcde" before "efgh"
+    # NB: always returns a lowercase string!
+    return True if sorted([x.lower(), y.lower()])[0] == x.lower() else False
```

Comment:
`return x.lower() < y.lower()`?

Reply:
Thanks for the quick feedback, will have a look at your remarks.

I wonder what was the case that dragged me into this specific rabbit hole though. The comments are from some earlier attempt and I wonder how I did not end up with the simple solution you proposed. :facepalm: :shrug: